ansible-ipi-install/roles/scale-worker/tasks/main.yml

```diff
@@ -25,6 +25,7 @@
     var: approve_csr_result
 
 - include_tasks: 10_bmc_secrets.yml
+  when: not redfish_inspection | bool
```

Comment:
Where do you propose this `redfish_inspection` boolean is set? I don't see it templated anywhere.